src/group-configurations/content-groups-section/index.jsx

```diff
@@ -60,6 +111,11 @@ ContentGroupsSection.propTypes = {
     scheme: PropTypes.string,
     version: PropTypes.number,
   }).isRequired,
```

Comment:
![image](https://github.com/raccoongang/frontend-app-course-authoring/assets/93188219/da397849-f52b-4353-a774-c347ab5ee98e)

Reply:
moved to contants